docs/examples/language_model/language_model2.md

```diff
@@ -0,0 +1,292 @@
+# Train your own LSTM based Language Model
```

Comment:
Could you rename them to:
- `language_model/language_model.md ` -> `language_model/use_pretrained_lm.md `
- `language_model/language_model2.md ` -> `language_model/train_language_model.md `